aten/src/ATen/cudnn/Descriptors.cpp

```diff
@@ -110,7 +110,7 @@ void FilterDescriptor::set(const at::Tensor &t, int64_t pad) {
     throw std::runtime_error("cuDNN supports only up to " STR(CUDNN_DIM_MAX) " dimensions");
 #undef _STR
 #undef STR
-  if (!t.is_contiguous(t.suggest_memory_format())) {
+  if (!t.is_contiguous(t.suggest_memory_format(/*channels_last_strides_exact_match=*/true))) {
```

Comment:
Same as the comment below. (But this would be less important, as N111 as data input is rare?)